Update enable-the-report-message-add-in.md (#5795)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/enable-the-report-message-add-in.md

- For more information on how to report a message using the Report Message feature, see [Report false positives and false negatives in Outlook](report-false-positives-and-false-negatives.md).

> - For organizations that have a URL filtering or security solution in place (i.e. Proxy, FireWall, etc) you will need to have ipagave.azurewebsites.net and outlook.office.com endpoints allowed to be reached on HTTPS protocol.

Reviewer: ```suggestion
- Organizations that have a URL filtering or security solution (such as Proxy and FireWall)  in place will need to have ipagave.azurewebsites.net and outlook.office.com endpoints allowed to be reached on HTTPS protocol.
```
